.github/workflows/apt-packages.yaml

```diff
@@ -4,6 +4,7 @@ on:
   schedule:
     # run daily 0:00 on main branch
     - cron: '0 0 * * *'
+  pull_request:
```

Comment:
Do we want to run this test for PRs?

Reply:
no this was just for testing that the change works, will remove before merging